crates/rpc/rpc-eth-types/src/cache/multi_consumer.rs

```diff
@@ -110,6 +127,7 @@ where
             cache: LruMap::new(ByLength::new(max_len)),
             queued: Default::default(),
             metrics: CacheMetrics::new_with_labels(&[("cache", cache_id.to_string())]),
+            memory_usage: CacheMetrics::new_with_labels(&[("cache", cache_id.to_string())]),
```

Comment:
could we make this a new field instead:

https://github.com/paradigmxyz/reth/blob/2e4376f35904e00b80926adf9954f50a051881a6/crates/rpc/rpc-eth-types/src/cache/metrics.rs#L8-L8

and also add a new grafana entry, ref: https://github.com/paradigmxyz/reth/blob/2e4376f35904e00b80926adf9954f50a051881a6/etc/grafana/dashboards/overview.json#L8268-L8268